Evaluator/TA/trend_evaluator/trend.py

```diff
@@ -27,6 +27,80 @@
 import tentacles.Evaluator.Util as EvaluatorUtil
 
 
+class SuperTrend(evaluators.TAEvaluator):
```

Comment:
Can you call it `SuperTrendEvaluator` please? Because this is an Evaluator 😉

Reply:
done